src/storage/mod.rs

```diff
@@ -69,13 +69,29 @@ pub enum Mutation {
 #[allow(clippy::match_same_arms)]
 impl Mutation {
     pub fn key(&self) -> &Key {
-        match *self {
+        match self {
```

Comment:
why remove *

Reply:
Because we do not need it.

Reply:
It's just unnecessary after an ergonomics improvement.
https://github.com/rust-lang/rfcs/blob/master/text/2005-match-ergonomics.md